cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -85,7 +96,7 @@ def _run_task(self):
 class KeywordLogger(object):
     ROBOT_LISTENER_API_VERSION = 2
 
-    def start_keyword(name, attrs):
+    def start_keyword(self, name, attrs):
```

Comment:
Hmm, oops. But why didn't this break before?

Reply:
It wasn't broken, actually, just confusing and non-idiomatic. 

You passed a class and it treated the class as an object. The self pointer was never needed and the class was never instantiated into an object.

I left it the way it was at first but it confused Bryan. And now it confused you (the author). And it had confused me (briefly) at first too. So now it is more idiomatic Python even though the old way worked okay.

So do you think that it was just an accident that it worked? Did you intend for it to be used as a traditional class instead of just a namespace with a function in it?

Reply:
Just an accident, I think. I may have copied code from some example of RF listeners.